rollup/internal/controller/sender/sender.go

```diff
@@ -619,12 +619,13 @@ func makeSidecar(blob *kzg4844.Blob) (*gethTypes.BlobTxSidecar, error) {
 	var proofs []kzg4844.Proof
 
 	for _, b := range blobs {
-		c, err := kzg4844.BlobToCommitment(b)
+		bCopy := b
```

Comment:
because of the following `golint` check:
```
internal/controller/sender/sender.go:622:38: G601: Implicit memory aliasing in for loop. (gosec)
                c, err := kzg4844.BlobToCommitment(&b)
                                                   ^
internal/controller/sender/sender.go:627:38: G601: Implicit memory aliasing in for loop. (gosec)
                p, err := kzg4844.ComputeBlobProof(&b, c)
                                                   ^
```
I have checked that this loop does not have race conditions (no concurrency inside the loop) but changed the implementation to pass this check.

Reply:
refactored the code to remove copy and pass `golint` check, in [f430a8e](https://github.com/scroll-tech/scroll/pull/1298/commits/f430a8e98b7c6fbff3923a91233f0df41b77bc87)